components/CheckoutWizard/CheckoutWizard.tsx

```diff
@@ -15,10 +15,11 @@ const ContentContainer = styled.div`
   width: 50%;
 `
 type Props = {
-  children?: ReactNode
+  children?: ReactNode,
```

Comment:
Not sure if I agree after the type changes with React 18, but no big deal for me

Reply:
what type changes are you referring to? afaik they deprecated `VoidFunctionComponent`, removed `children` from `FunctionComponent` and made it preferable to use `FunctionComponent<PropsWithChildren<T>>`

Reply:
Since React are treating children as ordinary props like all others props I prefer to just type them 🤷‍♀️ Could also be more or less strict according to the specific use case. Let me see if I can find one of the sources I read back when I was doing this for Envis.